Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -343,7 +343,7 @@ public struct CustomerCenterConfigData: Equatable {
                     openMethod: OpenMethod? = nil,
                     type: PathType,
                     detail: PathDetail?,
-                    refundWindowDuration: RefundWindowDuration?) {
+                    refundWindowDuration: RefundWindowDuration? = .forever) {
```

Comment:
I think this is correct. The refund window should be nil. It should only not be nil when it's a refund path, or at least I think that's how defined in the backend